regression/scripts/Vehicles/Tiltwing.py

```diff
@@ -388,32 +395,92 @@ def configs_setup(vehicle):
+    config.propulsors.vectored_thrust.pitch_command   = 3.  * Units.degrees 
+    configs.append(config)
+
+    # ------------------------------------------------------------------
+    #   Hover-to-Cruise Configuration
+    # ------------------------------------------------------------------
+    config                                            = SUAVE.Components.Configs.Config(base_config)
+    config.tag                                        = 'transition_seg_2_3'
+    vector_angle                                      = 15.0  * Units.degrees  
+    config.propulsors.vectored_thrust.thrust_angle    = vector_angle
+    config.wings.main_wing.twists.root                = vector_angle
+    config.wings.main_wing.twists.tip                 = vector_angle
+    config.wings.canard_wing.twists.root              = vector_angle
+    config.wings.canard_wing.twists.tip               = vector_angle 
+    config.propulsors.vectored_thrust.pitch_command   = 5.  * Units.degrees     
+    configs.append(config)
+
     # ------------------------------------------------------------------
     #   Cruise Configuration
     # ------------------------------------------------------------------
     config                                            = SUAVE.Components.Configs.Config(base_config)
-    config.tag                                        = 'cruise' 
-    config.propulsors.vectored_thrust.thrust_angle    =  0. * Units.degrees
+    config.tag                                        = 'cruise'   
+    vector_angle                                      = 0.0 * Units.degrees
+    config.propulsors.vectored_thrust.thrust_angle    = vector_angle
+    config.wings.main_wing.twists.root                = vector_angle
+    config.wings.main_wing.twists.tip                 = vector_angle
+    config.wings.canard_wing.twists.root              = vector_angle
+    config.wings.canard_wing.twists.tip               = vector_angle  
     config.propulsors.vectored_thrust.pitch_command   = 10.  * Units.degrees  
-    configs.append(config)  
-    
+    configs.append(config)    
+
+
+
+    # ------------------------------------------------------------------
+    #   Hover Configuration
+    # ------------------------------------------------------------------
+    config                                            = SUAVE.Components.Configs.Config(base_config)
+    config.tag                                        = 'hover_descent'
+    vector_angle                                      = 90.0  * Units.degrees  
+    config.propulsors.vectored_thrust.thrust_angle    = vector_angle
+    config.wings.main_wing.twists.root                = vector_angle
+    config.wings.main_wing.twists.tip                 = vector_angle
+    config.wings.canard_wing.twists.root              = vector_angle
+    config.wings.canard_wing.twists.tip               = vector_angle     
+    config.propulsors.vectored_thrust.pitch_command   = -5.  * Units.degrees     
```

Comment:
we found during the last release you need to set the propeller pitch command not the propulsors pitch command

Reply:
It is linked (similar to the thrust angle). Check lines 138-140. Issue Resolved